Formula/open-sp.rb

```diff
@@ -12,14 +12,18 @@ class OpenSp < Formula
     sha256 "d7b79be390f3c2b2a823e1156d896200db397dffb6cb6e6712d27539e05ca18b" => :yosemite
   end
 
+  depends_on "docbook" => :build
+  depends_on "xmlto" => :build
+  depends_on "ghostscript" => :build
+
   def install
     system "./configure", "--disable-debug", "--disable-dependency-tracking",
                           "--prefix=#{prefix}",
                           "--mandir=#{man}",
-                          "--disable-doc-build",
                           "--enable-http",
                           "--enable-default-catalog=#{etc}/sgml/catalog",
```

Comment:
The build failure is caused by the comma at the end of this line.

Reply:
not enough coffee, thanks for the catch